packages/userstorage/src/browser/user-storage-frontend-module.ts

```diff
@@ -21,6 +21,7 @@ import { UserStorageServiceFilesystemImpl } from './user-storage-service-filesys
 import { UserStorageService } from './user-storage-service';
 
 export default new ContainerModule(bind => {
-    bind(ResourceResolver).to(UserStorageResolver).inSingletonScope();
+    bind(UserStorageResolver).toSelf().inSingletonScope();
+    bind(ResourceResolver).toService(UserStorageResolver);
```

Comment:
An idea was that `UserStorageResolver` is based on `UserStorageService` and extensions can rebind `UserStorageService` to provide an alternative implementation.

Reply:
Thanks!